docs/general/dql/selects.rst

```diff
@@ -485,6 +487,14 @@ does always return ``NULL`` when comparing ``NULL``.
     +----------+
     SELECT 1 row in set (... sec)
 
+.. CAUTION::
+
+    ``IS NULL`` clauses can slow a query down because CrateDB has to iterate
+    over all rows for the comparison and cannot utilize the index.
+
+    For better performance, consider using :ref:`fulltext search
```

Comment:
Not sure if this makes sense here in this context to advocate using a fulltext search. Not sure how that would be used as an alternative to `IS NULL`

Reply:
was basing this off of the comments on the parent issue. figured that a fulltext search on an empty string might be what was being suggested